src/dex-ui/services/HederaService/HederaService.ts

```diff
@@ -202,23 +202,10 @@ function createHederaService() {
     return proposalTransactionResponse;
   };
 
-  const castVote = async ({ proposalId, voteType, signer }: any) => {
-    const contractFunctionParameters = new ContractFunctionParameters().addUint256(proposalId).addUint8(voteType);
-    const castVoteTransaction = await new ContractExecuteTransaction()
-      .setContractId(GOVERNOR_PROXY_CONTRACT.ContractId)
-      .setFunction(GovernorContractFunctions.CastVote, contractFunctionParameters)
-      .setGas(900000)
-      .setNodeAccountIds([new AccountId(3)])
-      .freezeWithSigner(signer);
-
-    const response = await castVoteTransaction.executeWithSigner(signer);
-    return response;
-  };
-
   const fetchProposalState = async (proposalId: BigNumber): Promise<BigNumber | undefined> => {
```

Comment:
These functions will need to be updated to handle dynamic proposal contract ids.